src/main.py

```diff
@@ -190,13 +190,14 @@ def _createEvalProcess(components, variables):
   unique = list(OrderedDict.fromkeys(ordered))
   return unique
 
-def componentLifeCashflow(comp, cf, variables, lifetimeCashflows, v=100):
+def componentLifeCashflow(comp, cf, variables, lifetimeCashflows, v=100, pyomoSwitch=None):
   """
     Calcualtes the annual lifetime-based cashflow for a cashflow of a component
     @ In, comp, CashFlows.Component, component whose cashflow is being analyzed
     @ In, cf, CashFlows.CashFlow, cashflow who is being analyzed
     @ In, variables, dict, RAVEN variables as name: value
     @ In, v, int, verbosity
+    @ In, pyomoSwitch, pyo.ConcreteModel(), concrete model including all variables
```

Comment:
can this be a boolean value that defaults to `False` or do we need the model?

Also, docstring needs the "optional" tag.

Reply:
Will change type and docstring for all similar comments below.